### PR TITLE
Merge revision into develop

### PR DESCRIPTION
**Pull request by DavidMansolino**
conflicts:
```
	modifié des deux côtés :  src/install_scripts/qt_linux_installer.sh
	modifié des deux côtés :  src/packaging/files_core.txt
	modifié des deux côtés :  src/packaging/webots_distro.c
```

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

File: src/packaging/webots_distro.c [hunk 1]
```diff
@@ -1084,30 +1084,6 @@ static void create_file(const char *name, int m) {
       fprintf(fd, "mkdir usr/local/bin\n");
       fprintf(fd, "ln -s /usr/local/%s/webots usr/local/bin/webots\n", application_name_lowercase_and_dashes);
       fprintf(fd, "cd %s/debian/usr/local/%s/lib/webots\n", distribution_path, application_name_lowercase_and_dashes);
```
Comment (omichel) [inline src/packaging/webots_distro.c]: This line could be removed as well.

Comment (DavidMansolino) [reply src/packaging/webots_distro.c]: It will be removed at the next merge.

**Comment by DavidMansolino** [comment]
:+1:  (updated with revision)

**Pull request merged**